plugins/inputs/kube_inventory/README.md

```diff
@@ -190,10 +190,10 @@ subjects:
   - tags:
     - node_name
   - fields:
-    - capacity_cpu_cores
```

Comment:
Removing fields breaks backward compatibility so we won't be able to merge this as is.

For example, if a user of this plugin saves metrics to a database and uses the capacity_cpu_cores field, then they upgrade to a telegraf that doesn't send the capacity_cpu_cores field, their queries will stop returning what they expected.

Adding new fields is ok.  It would also be ok to add a setting to let the user choose whether they want whole cpu or millicpu units and defaulted the setting to the previous behavior of using whole cpu units.

Reply:
Sounds good - I've added back the original fields.